tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCLDRFile.java

```diff
@@ -195,7 +195,7 @@ public void testExtraPaths() {
         Counter<String> extraPaths = new Counter<>();
         for (String locale : sdi.hasGrammarInfo()) {
             if (sdi.getGrammarInfo(locale).hasInfo(GrammaticalTarget.nominal)) {
-                final CLDRFile cldrFile = CLDRConfig.getInstance().getCldrFactory().make(locale, true);
+                final CLDRFile cldrFile = CLDRConfig.getInstance().getFullCldrFactory().make(locale, true);
```

Comment:
If there is grammar info, check it, even if it's in seed. (there's no seed/supplemental/* )